modules/database/variables.tf

```diff
@@ -0,0 +1,15 @@
+variable "database_name" {
+    description = "The name of the database"
```

Comment:
We could use Like glasslenses_database

Reply:
The reason I left so generic is because we could want to change the database later or reuse the modules for something else

Reply:
OK.We can keep for know. I got the logic .Thanks